spec/decorators/has_hint_option_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe HasHintOption do
+  subject { described_class.new(base_input) }
+  let(:object) { build_object }
+  let(:base_input) { build_base_input }
+  let(:validators) { {} }
+  let(:property) { :lcsubject }
+
+  describe "#hint" do
+    context "when there is no validator" do
+      it "should return nil" do
+        expect(subject.hint).to eq nil
+      end
+    end
+    context "when there is a validator" do
+      let(:validators) do
+        {
+          property => [validator]
+        }
+      end
+      let(:validator) { build_validator }
+
+      it "should return that validator's message" do
+        expect(subject.hint).to eq "Must fulfill: be awesome"
+      end
+    end
+  end
+
+  def build_validator
```

Comment:
Is there any particular reason you chose methods over just defining this stuff in the `let` blocks?  Seeing `def` seems out of place within our test suite.